### PR TITLE
NL Feedback: log chart and query IDs

### PR DESCRIPTION
**Pull request by pradh**
Changes:
1. Log chart IDs computed from dom ID
2. Support feedback at query-level
3. Move the thumbs-down to the right

Validation:

Screenshot 2023-06-26 at 5 01 57 PM

Screenshot 2023-06-26 at 5 30 58 PM

### CODE REVIEW

**Review by jehangiramjad — APPROVED** [review]

File: static/js/utils/nl_interface_utils.ts [hunk 1]
```diff
@@ -28,6 +30,36 @@ const VERSION_PARAM_PREFIX = "&entry.1420739572=";
...
+  queryIdx: number;
+  categoryIdx: number;
+  blockIdx: number;
+  columnIdx: number;
+  tileIdx: number;
+}
+
+// Given the dom ID of a chart, extract its relevant indexes in
+// the query session and SubjectPageConfig proto.
+export function getNLChartId(idStr: string): ChartId {
```
Comment (shifucun) [inline static/js/utils/nl_interface_utils.ts:47]: Might need some integration test to make sure the id format does not change

File: static/js/components/tiles/chart_tile.tsx [hunk 2]
```diff
@@ -25,6 +25,10 @@ import { ASYNC_ELEMENT_HOLDER_CLASS } from "../../constants/css_constants";
 import { INITAL_LOADING_CLASS } from "../../constants/tile_constants";
 import { ChartEmbed } from "../../place/chart_embed";
 import { NlSessionContext } from "../../shared/context";
+import {
+  CHART_FEEDBACK_SENTIMENT,
+  getNLChartId,
```
Comment (shifucun) [inline static/js/components/tiles/chart_tile.tsx:30]: nit s/getNL/getNl to be consistent.

**Review by shifucun — APPROVED** [review]
very nice!

File: static/js/components/tiles/chart_tile.tsx (hunk 2 above)
Comment (pradh) [reply static/js/components/tiles/chart_tile.tsx]: Done.

File: static/js/utils/nl_interface_utils.ts (hunk 1 above)
Comment (pradh) [reply static/js/utils/nl_interface_utils.ts]: Added a TODO and it will be part of webdriver test like we discussed.

**Review by pradh — COMMENTED** [review]
Thanks for the review!

**Pull request merged**